### ISSUE TITLE
UTC issue in database

### ISSUE THREAD

**Issue by todriguez** [flackdl/recipe_api#17]
I have tried making sure the postgresql database settings are UTC and the settings.py but I am not sure what else to do. 

recipme/recipe_api/myenv/lib/python3.11/site-packages/django/db/backends/postgresql/utils.py", line 6, in utc_tzinfo_factory
    raise AssertionError("database connection isn't set to UTC")
AssertionError: database connection isn't set to UTC

**Comment by flackdl** [comment]
Well you're the first person to show interest in this!  Let's roll up our sleeves and figure it out.

I believe this is an issue with the Postgres/python library called `psycopg2-binary`.  Let me investigate and I'll get back with you.

**Comment by flackdl** [comment]
@todriguez I updated the python depdendencies to support python 3.11 as I can tell that's what you're using.  I just had to upgrade Django & psycopg2 (the postgres client).

I tested it locally and it's working for me.

Let me know how it goes.

**Comment by flackdl** [comment]
Make sure you pull the updates and go through the steps again.

**Comment by todriguez** [comment]
I sorted it not long after I raised the issue. It did my head in for a while. I think I had to get the psycog binary instead. Now i've got it having scraped 4200 recipes, but when i spin up the webserver, though it looks nice, when i attempt to create a query it just flashes a response for a second then blank. 


Failed to load resource: the server responded with a status of 500 (Internal Server Error)

**Comment by flackdl** [comment]
You're right.  The query isn't working - I rushed the python 3.11 update and didn't test it end-to-end.  Let me get it updated properly and i'll keep you posted. 

In the meantime, see this demo and make sure it's what you're looking for in the 1st place.  

https://recipes.eerieemu.com/
https://recipes.eerieemu.com/api/
